src/test/osd/osd-scrub-repair.sh

```diff
@@ -19,17 +19,19 @@ source $CEPH_ROOT/qa/workunits/ceph-helpers.sh
 
 if [ `uname` = FreeBSD ]; then
     use_bluestore=false
-    termwidth=$(stty -a | head -1 | sed -e 's/.* \([0-9]*\) columns.*/\1/')
 else
     use_bluestore=true
     termwidth=$(stty -a | head -1 | sed -e 's/.*columns \([0-9]*\).*/\1/')
+    if [ -n "$termwidth" -a "$termwidth" != "0" ]; then 
+        termwidth="-W ${termwidth}" 
+    fi
+    DIFFCOLOPTS="-y $termwidth"
```

Comment:
for more discussions on this, see https://github.com/ceph/ceph/pull/9613#discussion_r86856656,